index.js

```diff
@@ -14,51 +14,42 @@ const boxenOptions = {
     backgroundColor: "#FFFFFF"
 }
 
-
-
-const version = boxen(chalk.redBright.bold("Urltester 1.0.0"), boxenOptions);
-
-if(process.argv.length == 2 ){
-    
+const messageOne = () => {
     console.log(chalk.red("\n                Standard user manual                "));
     console.log((chalk.gray("----------------------------------------------------"))); 
     console.log(chalk.redBright("urltester filename   -"), "reports good,bad,unknown urls"); 
     console.log(chalk.redBright("urltester v |version -"), "displays tool version"); 
     console.log((chalk.redBright("----------------------------------------------------\n"))); 
-}
+}; 
 
-else {
-    if(process.argv[2] === "v" || process.argv[2] === "version") {
-        console.log(version);
-    }
-    else {      
-        const filePath = path.join(__dirname,process.argv[2]);
-        fs.readFile(filePath,'utf-8', function(err, data) {
-            if(err) {
-                console.log(chalk.red("Unsuccesful to read file"), err)
-            }
-            else {
-                const urlArr = data.match(/(http|https)(:\/\/)([\w+\-&@`~#$%^*.=/?:]+)/gi);
+const version = boxen(chalk.redBright.bold("Urltester 1.0.0"), boxenOptions);
+
+if(process.argv.length == 2 ) messageOne();
+else if(process.argv[2] === "v" || process.argv[2] === "version" || process.argv[2] === "-v") {
```

Comment:
I also add the -v. 
And I suggest you to use only --v and/or -v. 
But it's just a suggestion.

Reply:
Yeah, does no harm to add -v as well